primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-es.js

```diff
@@ -71,11 +71,11 @@ if (window.PrimeFaces) {
       pageLabel: "Página {page}",
       firstPageLabel: "Primera Página",
       lastPageLabel: "Última Página",
-      nextPageLabel: "Siguiente Página",
+      nextPageLabel: "Página Siguiente",
       previousPageLabel: "Página Anterior",
       rowsPerPageLabel: "Filas por página",
-      jumpToPageDropdownLabel: "Ir al menú desplegable de página",
-      jumpToPageInputLabel: "Ir a la entrada de página",
+      jumpToPageDropdownLabel: "Menú desplegable para ir a una página",
+      jumpToPageInputLabel: "Entrada para ir a una página",
```

Comment:
Same here, maybe is "Ir a una pagina" or "Ir a la página X" because the variable name is "jumpToPage". I cannot see the context @melloware

Reply:
Its for the paginator:

![image](https://github.com/primefaces/primefaces/assets/4399574/6662e106-fd92-4dec-85c5-cc4a92308c4a)